app/views/pages/home.html.erb

```diff
@@ -1,2 +1,53 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="home-page container">
+    <div class="text-center">
+        <h4>Book unique places to stay and things to do.</h4>
+    </div>
+    <%= form_for :search do |f| %>
+        <div class="row mb-5 mt-5">
+            <input class="form-control mr-sm-2 col-5" type="search" placeholder="Where are you going?" aria-label="Search">
+            <%= f.submit "Strat Date", class: "input-wid form-control col-2 ml-4" %>
+            <%= f.submit "End Date", class: "input-wid form-control col-2 ml-4" %>
+            <%= f.submit "Serach", class: "input-wid form-control col-2 ml-4 btn-info" %>
+        </div>
+    <% end %>
+    <div class="mt-5">
+    <h1>Homes</h1>
+        <div class="card-deck row row-cols-3">
+            <% @rooms.each do |room| %>
+                <div class="col mb-3">
+                    <div class="card h-100">
+                        <%= image_tag room.photos.first.image%>
+                        <div class="card-body">
+                            <%= link_to room.listing_name %><br>
+                            $<%= room.price %> - <%= room.room_type %> - <%= room.bedroom_count %> Betrooms<br>
+                            <%#= stars %>                        
+                            <%= form_for @review do |f| %>
+                                <%= f.hidden_field :reservation_id, value: room.id %>
+                                <%= f.hidden_field :room_id, value: room.id %>
+                                <div id="stars_<%= room.id %>"></div>                                                
+                                <script>
+                                    $('#stars_<%= room.id %>').raty({
+                                        path: '/assets',
+                                        scoreName: 'review[rating]',
+                                        score: 0
+                                    });
+                                </script>
+                            <% end %>
```

Comment:
You can remove this since we are not going to create a review on this page.
No need to display the stars on this task for now.